migrations/20240703182015_user_device_tableland_id.sql

```diff
@@ -0,0 +1,12 @@
+-- +goose Up
+-- +goose StatementBegin
+SET search_path = devices_api, public;
+ALTER TABLE user_devices ADD COLUMN definition_id text not null default '';
```

Comment:
let's make it nullable to start, that way don't need to default to a blank string for now. Or was your thought to backfill and then save this step? We'd probably want to remove the default '' after doing that, like either way i think we'll need a migration after backfilling